src/Artsy/Router/index.ts

```diff
@@ -11,11 +12,11 @@ export { ContextProvider, ContextConsumer } from "Artsy/SystemContext"
 /**
  * Configuration used when creating a new Router app
  */
-export interface RouterConfig {
+export interface RouterConfig<E extends Environment = Environment> {
```

Comment:
Why extend `Environment` here?

Reply:
Not sure too sure! Was just curious. Having `E extends Environment` seems to imply that there's something more complex going on somewhere when really its just a setup issue for the type-checker.

Reply:
Cleaned this up.